pkg/ngrokgodriver/driver.go

```diff
@@ -0,0 +1,121 @@
+package ngrokgodriver
+
+import (
+	"context"
+	"io"
+	"log"
+	"net"
+	"reflect"
+
+	"golang.ngrok.com/ngrok"
+	"golang.ngrok.com/ngrok/config"
+	"golang.org/x/sync/errgroup"
+)
+
+type TunnelManager struct {
```

Comment:
Nit: we should probably doc our exported functions in the pkg folder? If that's something we all think we should do, maybe in a subsequent pr we can setup a linter

Reply:
Good catch. I'll add some docs to these